lynx/float.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // EncryptedFloat represents a float that is encryptable as a string.
 type EncryptedFloat struct {
-	str *string
+	Str *string
```

Comment:
Lets rename this to `StringValue` or similar so it's more obvious what it is - hard to change later on once it's exported.